templates/database.cfn.yml

```diff
@@ -177,6 +187,14 @@ Resources:
       Description: Setup parameters for Queensland Government databases
       Family: !Sub "${DBEngine}${DBEngineVersion}"
       Parameters:
-        datestyle: "ISO, DMY"
-        rds.adaptive_autovacuum: 1
-        rds.force_ssl: 0
+        Fn::If:
```

Comment:
Probably simpler to convert to a Jinja template and use Jinja syntax to control this.

Reply:
this is a standard cfn template, could be j2 but it works.